src/ontology/mondo-edit.obo

```diff
@@ -143956,11 +143956,13 @@ xref: Orphanet:618 {source="OMIM:155600"}
 xref: SCTID:254819008 {source="MONDO:equivalentTo"}
 xref: UMLS:C0151779 {source="MONDO:relatedTo", source="OMIM:155600"}
 xref: UMLS:C1835047 {source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset", source="OMIM:155600"}
-is_a: MONDO:0018453 {source="Orphanet:404560/btnt"} ! familial atypical multiple mole melanoma syndrome
-is_a: MONDO:0024462 {source="MONDO:OMIM", source="MONDO:Redundant", source="OMIM:155600"} ! familial cutaneous melanoma
-intersection_of: MONDO:0024462 ! familial cutaneous melanoma
+intersection_of: MONDO:0020573 ! inherited disease susceptibility
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/9588 ! PTEN
+intersection_of: predisposes_towards MONDO:0005012 ! cutaneous melanoma
+relationship: excluded_subClassOf MONDO:0018453 {source="Orphanet:404560/btnt"} ! familial atypical multiple mole melanoma syndrome
```

Comment:
if you reviewed this and agree with the Orphanet classification, can you add your ORCID too?
Otherwise this is is going to get flagged for review if there is only an ORDO source for the subclass assertion (and we'll have to review it again)

thanks!

Reply:
Oh, nevermind, I see this is excluded subclass of!

Ignore me!